docs/README.md

```diff
@@ -165,16 +165,17 @@ send data points.
+|SPLUNK_REALM|`us0`| | The name of your organization's realm as described [here](https://dev.splunk.com/observability/docs/realms_in_endpoints/). It is used to build a standard endpoint for ingesting metrics.                                                                               |
+|SPLUNK_INGEST_URL| |`https://ingest.eu0.signalfx.com`| Real-time Data Ingest - you can find it in your account settings screen. It overrides the endpoint defined by the `SPLUNK_REALM` variable and it can be used to point to non standard endpoints.                                                                        |
+|SPLUNK_ACCESS_TOKEN| | | Access token as described [here](https://docs.signalfx.com/en/latest/admin-guide/tokens.html#access-tokens).                                                                                                                                                            |
+|FAST_INGEST|`true`|`true` or `false`| Determines the strategy used to send data points. Use `true` to send metrics on every Lambda invocation ([fast ingest](#Fast-ingest) mode). With `false` metrics will be buffered and send out on intervals defined by `REPORTING_RATE` ([buffering](#Buffering) mode). |
+|REPORTING_RATE|`15`|An integer (seconds). Minimum value is 1s.| Specifies how often data points are sent to Splunk. Due to the way the AWS Lambda execution environment works, metrics may be sent less often.                                                                                                                          |  
+|REPORTING_TIMEOUT|`5`|An integer (seconds). Minimum value is 1s.| Specifies metric send operation timeout.                                                                                                                                                                                                                                |
+|VERBOSE|`false`|`true` or `false`| Enables verbose logging. Logs are stored in a CloudWatch Logs group associated with a Lambda Function.                                                                                                                                                                  |
+|HTTP_TRACING|`false`|`true` or `false`| Enables detailed logs on HTTP calls to Splunk.                                                                                                                                                                                                                          |
+|INSECURE_SSL|`false`|`true` or `false`| Enables skip check validation ssl on HTTP calls to Splunk.                                                                                                                                                                                             |
```

Comment:
```suggestion
|INSECURE_HTTPS|`false`|`true` or `false`| Enables skip certificate validation for HTTPS calls to Splunk.                                                                                                                                                                                             |
```

Is it a call to Splunk Observability Cloud or Splunk Enterprise? AFAIK it is to Splunk Observability Cloud.